src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -1245,5 +1245,40 @@ private void TestNormalize(SyntaxTriviaList trivia, string expected)
             var actual = trivia.NormalizeWhitespace("    ").ToFullString().NormalizeLineEndings();
             Assert.Equal(expected.NormalizeLineEndings(), actual);
         }
+
+        [Fact]
+        [WorkItem(60884, "https://github.com/dotnet/roslyn/issues/60884")]
+        public void TestNormalizeXmlArgumentsInDocComment1()
+        {
+            const string Expected = @"/// Prefix <b a=""x"" b=""y"">S_OK</b> suffix";
+            const string Text = @"/// Prefix <b    a=""x""  b=""y"" >S_OK</b> suffix";
+            TestNormalizeDeclaration(Text, Expected);
+        }
+
+        [Fact]
+        [WorkItem(60884, "https://github.com/dotnet/roslyn/issues/60884")]
+        public void TestNormalizeXmlArgumentsInDocComment2()
+        {
+            const string Expected = @"/// Prefix <b a=""x"" b=""y"">S_OK</b> suffix";
+            TestNormalizeDeclaration(Expected, Expected);
+        }
+
+        [Fact]
+        [WorkItem(60884, "https://github.com/dotnet/roslyn/issues/60884")]
+        public void TestNormalizeXmlArgumentsInDocComment3()
+        {
+            const string Expected = @"/// Prefix <b a=""x"" b=""y"" /> suffix";
+            const string Text = @"/// Prefix <b a=""x"" b=""y"" /> suffix";
```

Comment:
>  [](http://example.com/codeflow?start=55&length=1)

Is it syntactically valid to omit the space between the XML attributes? Consider adding a test.

Reply:
No, it's not, at least not for all XML parsers. I actually need this fix in Roslyn because `DocumentationCommentCompiler` failed on generated Roslyn syntax trees after normalization. However, I added 2 tests to show normalization can now fix issues in XML trees such as the missing spaces between the attributes.